### ISSUE TITLE
Ugly fonts after auto update on debian 9

### ISSUE THREAD

**Issue by mundusnine** [snapcrafters/vscode#9]
I had the revision 50(vscode 1.25) and when it updated to revision 51(vscode 1.26.1) I got the ugly fonts. 
snap --version gives:
```
snap    2.34.3
snapd   2.34.3
series  16
debian  9
kernel  4.9.0-8-amd64
```

In apt snapd version is : 2.21-2+b1

I tried reinstalling the snap with no success. When installed with flatpak no issue with the fonts.
![image](https://user-images.githubusercontent.com/29337013/44596350-c3b42c80-a799-11e8-98e3-46ea4d1ab013.png)

**Comment by jjangga0214** [comment]
Same issue on ubuntu 18.10.

**Comment by flexiondotorg** [comment]
Fixed via #40
